### PR TITLE
Fixed build problems

### PR DESCRIPTION
**Pull request by dpgraham**
* An update to dependencies broke unit tests, so did some refactoring. (I still haven't figured out how to run the Java unit tests from a command line, so if somebody wants to checkout this branch and try something, that would be greatly appreciated)
* Made some Codacy fixes

### CODE REVIEW

File: espresso-server/app/src/androidTest/java/io/appium/espressoserver/EspressoServerRunnerTest.java [hunk 1]
```diff
@@ -34,22 +32,24 @@
...
     private final Server espressoServer = Server.getInstance();
 
     @Test
     public void startEspressoServer() throws InterruptedException, IOException, DuplicateRouteException {
-        try {
-            espressoServer.start();
-
-            while (!espressoServer.isStopRequestReceived()) {
-                Thread.sleep(1000);
+        System.out.println(System.getProperty("UNITTEST"));
```
Comment (mykola-mokhnach) [inline espresso-server/app/src/androidTest/java/io/appium/espressoserver/EspressoServerRunnerTest.java]: This is not a good thing to put println into a production code

File: espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/Rect.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
+package io.appium.espressoserver.lib.helpers;
+
+/**
+ * Copies parts of the Android Rect
+ *
+ * This is copied so that we can run in Java context
+ */
+public class Rect {
```
Comment (mykola-mokhnach) [inline espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/Rect.java]: cannot we just use roboelectric instead?

File: espresso-server/app/src/androidTest/java/io/appium/espressoserver/EspressoServerRunnerTest.java (hunk 1 above)
Comment (dpgraham) [reply espresso-server/app/src/androidTest/java/io/appium/espressoserver/EspressoServerRunnerTest.java]: That was  a mistake.

File: espresso-server/app/src/test/java/io/appium/espressoserver/test/assets/Helpers.java [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
     public static String readAssetFile(String filename) throws IOException {
```
Comment (mykola-mokhnach) [inline espresso-server/app/src/test/java/io/appium/espressoserver/test/assets/Helpers.java]: Cannot we move these files to test resources instead?

**Pull request merged**